README.md

```diff
@@ -48,9 +48,9 @@ npm install --save-dev redux-devtools
 
 ### The Gist
 
-The whole state of your app is stored in an object tree inside a single *store*.  
-The only way to change the state tree is to emit an *action*, an object describing what happened.  
-To specify how the state tree is transformed by the actions, you write pure *reducers*.
+The whole state of your app is retained in a single object that we will call *the store*.  
+The only way to alter the store is to emit an *action*, an object describing what is happening.  
+To specify how an action transforms the value of the store into another value, you write *reducers*, and these should be stateless (or "pure", or "deterministic", or "no-surprise") functions.
```

Comment:
We never say “value of the store” so this also goes out of sync with the rest of the docs. I think we can keep this section short (without clarifying purity) because it explored in great detail here: gaearon.github.io/redux/docs/basics/Reducers.html